spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -38,4 +38,21 @@
       expect(response).to redirect_to('/unauthorized')
     end
   end
+
+  describe "#accept_terms_of_services" do
+    let(:user) { create(:user) }
+
+    before do
+      allow(controller).to receive_messages spree_current_user: user
+      user.spree_roles << Spree::Role.find_or_create_by(name: 'admin')
+    end
+
+    it "updates terms_of_service_accepted_at" do
+      expect do
+        spree_post :accept_terms_of_services, id: user.id
+      end.to change { user.reload.terms_of_service_accepted_at }
```

Comment:
I prefer to have the reload in the action block to avoid reloading twice (before and after the action block). This is micro-optimisation though, sorry.
```suggestion
      expect do
        spree_post :accept_terms_of_services, id: user.id
        user.reload
      end.to change { user.terms_of_service_accepted_at }
```

Reply:
Don't be sorry, it's good to know :+1: